mgmt/rest/v2/api.go

```diff
@@ -56,25 +56,302 @@ func New(wg *sync.WaitGroup, killChan chan struct{}, protocol string) *apiV2 {
 
 func (s *apiV2) GetRoutes() []api.Route {
 	routes := []api.Route{
-		// plugin routes
+		// swagger:route GET /plugins plugins getPlugins
+		//
```

Comment:
No, comments have to stay like this format in order for them to show up in UI pretty. Otherwise, it shows up but not as pretty as it's.